number_theory/modular.h

```diff
@@ -256,6 +271,7 @@ std::ostream &operator<<(std::ostream &stream, const M &modular) {
 
 }  // namespace number_theory
 
+using number_theory::inverse;
```

Comment:
Do we want the function to be exported? If so, we need to give it a more descriptive name, such as `inverse_mod`.

Reply:
I think we should export it and I agree that the name is not clear enough. I was already thinking like "are we going to have another inverse in number_theory?", "maybe we should have used a modular namespace like modular::inverse.` etc.